src/main/java/org/opentripplanner/middleware/otp/response/Place.java

```diff
@@ -0,0 +1,53 @@
+package org.opentripplanner.middleware.otp.response;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonInclude;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonPropertyOrder;
+
+import java.util.Date;
+import java.util.Set;
+
+/**
+ * Plan response, place information. Produced using http://www.jsonschema2pojo.org/
+ */
+@JsonInclude(JsonInclude.Include.NON_NULL)
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class Place {
+
+    @JsonProperty("name")
```

Comment:
Are all of these `@JsonProperty` annotations needed?

Reply:
It doesn't look like it. Have removed.